sssd_test_framework/utils/sssd.py

```diff
@@ -13,13 +13,17 @@
 from ..hosts.base import BaseDomainHost
 from ..hosts.client import ClientHost
 from ..misc import to_list
+from ..roles.ad import AD
```

Comment:
@pbrezina I figured out this works, but I don't understand why. 

I had to split the imports, IPA and Samba ```if TYPE_CHECKING``` and the others without.